pkg/exporter/process.go

```diff
@@ -29,6 +30,9 @@ import (
 )
 
 const startTemplateID uint16 = 255
+const defaultCheckConnInterval = 10 * time.Second
+
+var oneByte = make([]byte, 1)
```

Comment:
why global variable?

Reply:
I don't want to recreate it every time when testing the connection. Moved it to ExportingProcess struct then.